apps/alert_processor/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule AlertProcessor.Mixfile do
   def project do
     [
       app: :alert_processor,
-      version: "0.0.59",
+      version: "0.0.1",
```

Comment:
suggestion: Would it make sense to bump these to `1.0.1` or something so that they are increasing?

Reply:
Ah, I should set it to `0.1.0`, since that's the default version when you generate a new Elixir project (and so it's what most of our other apps have).